networking/init-cms-bundle/4.0/config/packages/networking_init_cms.yaml

```diff
@@ -0,0 +1,87 @@
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_block.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_core.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_doctrine_admin.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_formatter.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_media.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_notification.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_user.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/stof_doctrine_extensions.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/symfony_cmf_routing_extra.yaml" }
+# Default security configuration
+#    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/security.yaml" }
+
+# Set up the serializer to read config file for App Page and User entities
+jms_serializer:
+    metadata:
+        directories:
+            NetworkingSonataMediaBundle:
+                namespace_prefix: "Sonata\\MediaBundle"
+                path: "@NetworkingInitCmsBundle/Resources/config/serializer"
+            app:
+                namespace_prefix: "App"
+                path: "%kernel.project_dir%/config/serializer"
+
+
+networking_init_cms:
+    #Base classes to be used for the page and user entities
+    class:
+        page: "App\\Entity\\Page"
+        user: "App\\Entity\\User"
+
+    # Bootstrap some template settings to get you started
+    templates:
+        'app_single_column':
+            template: "@NetworkingInitCms/sandbox/page/one_column.html.twig"
+            name: "Single Column"
+            icon: "bundles/networkinginitcms/img/template_header_one_column.png"
+            zones:
+                - { name: header, class: 'col-md-12' }
+                - { name: main_content, class: 'col-md-12'}
+        'app_two_column':
+            template: "@NetworkingInitCms/sandbox/page/two_column.html.twig"
+            name: "Two Column"
+            icon: "bundles/networkinginitcms/img/template_header_two_column.png"
+            zones:
+                - { name: header , class: 'col-md-12'}
+                - { name: left , class: 'col-md-3'}
+                - { name: right , class: 'col-md-9'}
+
+    #Set a cookie in the fronted end for language. If disabled, the URL will be prefixed with the locale
+    #allow_locale_cookie: true
+
+    #Simple php based cache for caching the rendered HTML in the frontend
+    cache:
+        activate: true #enable for product, false in dev config
+    #    cache_time: 86400 #default value
+    #    cache_service: 'Networking\InitCmsBundle\Cache\PageCache' #default value
+
+    #Show admin toolbar when in the frontend
+    #admin_toolbar:
+    #    toolbar: true #Default is activated
```

Comment:
Could you remove these config in comments? 

Rather give a link to your readme where you explain and show all possible config options.

Reply:
Ok, no problem. Done